catalyst_voices/packages/internal/catalyst_voices_models/lib/src/document/definitions/single_line_https_url_entry_definition.dart

```diff
@@ -17,7 +17,10 @@ final class SingleLineHttpsURLEntryDefinition
     DocumentSchemaProperty<String> schema,
     String? value,
   ) {
-    return DocumentValidator.validateString(schema, value);
+    final stringValidationResult =
+        DocumentValidator.validateString(schema, value);
+    if (stringValidationResult.isInvalid) return stringValidationResult;
```

Comment:
How about using brackets? It helps to separate this `if` body from the unrelated line below

```suggestion
    if (stringValidationResult.isInvalid) {
      return stringValidationResult;
   }
```